ethereumkit/src/test/java/org/p2p/ethereumkit/EthereumAddressValidatorTest.kt

```diff
@@ -0,0 +1,28 @@
+package org.p2p.ethereumkit
+
+import org.junit.Test
+import org.p2p.ethereumkit.external.utils.EthereumUtils
+
+class EthereumAddressValidatorTest {
+
+    @Test
+    fun `validate addresses for Ethereum`() {
```

Comment:
split into two tests - one for valid addresses and one for invalid.
i suggest given-when-then naming for test naming